wad.md

```diff
@@ -42,7 +42,7 @@ Following the WAD header is the directory entry.
 | ---:| ----:| ------ | ------------------------------------------------- |
 |   0 |    1 | u8     | ECDSA signature length                            |
 |   1 |   83 |        | ECDSA signature of entry headers, padded with `0` |
-|  84 |    8 | u64    | xxHash checksum                                   |
+|  84 |    8 |        | xxHash checksum                                   |
```

Comment:
Since we now also have xxh3_64bits hashes, it could be nice to specify `xxHash64` or, to follow the "official" name, `xxHash's XXH64` (`xxHash` being the family of hashes, and `XXH64` the name of the individual variant).

The same applies to the the WAD v3 header.